libherokubuildpack/src/buildpack_output/background_printer.rs

```diff
@@ -0,0 +1,195 @@
+//! This module is responsible for the logic involved in the printing to output while
```

Comment:
Review order comment: Everything in this file is internal to the crate. It's about half comments.